### PR TITLE
recreated images for poseidon.md

### PR DESCRIPTION
**Pull request by Otsar-Raikou**
Fixed 3 images shown in low quality - i have recreated the 3 images - please check me to see that i have not made a mistake

### CODE REVIEW

**Review by jeremyfelder — APPROVED** [review]

File: docs/docs/icicle/primitives/poseidon.md [hunk 1]
```diff
@@ -16,7 +16,9 @@ Poseidon starts with the initialization of its internal state, which is composed
 
 This is done to prevent collisions and to prevent certain cryptographic attacks by ensuring that the internal state is sufficiently mixed and unpredictable.
 
-![Alt text](image.png)
+![First Board - 1redone](https://github.com/ingonyama-zk/icicle/assets/122266060/52257f5d-6097-47c4-8f17-7b6449b9d162)
```
Comment (jeremyfelder) [inline docs/docs/icicle/primitives/poseidon.md:19]: The "Alt text" portion should be a description of what the image conveys.

File: docs/docs/icicle/primitives/poseidon.md [hunk 2]
```diff
@@ -26,7 +28,8 @@ To generate a secure hash output, the algorithm goes through a series of "full r
 
 ### Full rounds
 
-![Alt text](image-1.png)
+![First Board - 2redone](https://github.com/ingonyama-zk/icicle/assets/122266060/72b54275-d5c5-4b69-b91a-820eed9990a5)
```
Comment (jeremyfelder) [inline docs/docs/icicle/primitives/poseidon.md:31]: Is there an image that includes the arrows from each previous stage to the next one?

**Review by jeremyfelder — CHANGES_REQUESTED** [review]
Minor changes

File: docs/docs/icicle/primitives/poseidon.md (hunk 1 above)
Comment (Otsar-Raikou) [reply docs/docs/icicle/primitives/poseidon.md:19]: That info is missing in the original as far as i can see

**Review by jeremyfelder — APPROVED** [review]

**Review by LeonHibnik — APPROVED** [review]

**Pull request merged**